ecflow_cloudwatch/ecflow_state_parser.py

```diff
@@ -129,13 +127,19 @@ def parse(self):
                                 continue
 
                             if self.is_meter(task):
-                                for meter in task.meters:
-                                    min, max, threshold = meter.min(), meter.max(), meter.value()
-                                available_data.append(','.join(map(lambda x: str(x), [min, max, threshold])))
+                                # Use fetch recent only for meter data 
+                                if self.fetch_new and  self.filter_date(task.get_abs_node_path()):
+                                    for meter in task.meters:
+                                        min, max, threshold = meter.min(), meter.max(), meter.value()
+                                        available_data.append(','.join(map(lambda x: str(x), [min, max, threshold])))
+                                else:
+                                    for meter in task.meters:
+                                        min, max, threshold = meter.min(), meter.max(), meter.value()
+                                        available_data.append(','.join(map(lambda x: str(x), [min, max, threshold])))
```

Comment:
This is exacly the same as line 132-134?

Reply:
Fixed!

Reply:
137 and 139 are still the same...

Reply:
Yes, due to old data sets I have tests, I have to add this but I removed it now and refactor test to check only output type